docs/modules/hive/partials/supported-versions.adoc

```diff
@@ -2,5 +2,6 @@
 // This is a separate file, since it is used by both the direct Hive-Operator documentation, and the overarching
 // Stackable Platform documentation.
 
-- 4.0.1 (LTS)
+- 4.0.1 (experimental)
```

Comment:
nit: It would be awesome if we could add a short sentence why we marked it as experimental.
Not blocking though

Reply:
done https://github.com/stackabletech/hive-operator/pull/582/commits/756b5a5cc0f8f0bccdf5fbcd1143924ba8dfb4ad